### ISSUE TITLE
Use some linter or git config to unify line endings

### ISSUE THREAD

**Issue by Duelsik** [sourcehold/Sourcehold#43]
Add configs/tools and renormalize codebase

**Comment by skrax** [comment]
I have added several sanitizers, clang-tidy to my [draft](https://github.com/sourcehold/Sourcehold/pull/45).
You can check the [cmake](https://github.com/skrax/Sourcehold/tree/skrax-master/cmake) directory and the main [CMakeLists.txt](https://github.com/skrax/Sourcehold/blob/skrax-master/CMakeLists.txt)for changes

**Comment by Duelsik** [comment]
Seen already, a bit shame that it is not separate PR, but we can live with it :)

**Comment by skrax** [comment]
Yes. It's a bit annoying, because of cxxopts and ghc_filesystem warnings.
I had to add diagnostic guards to skip over them. Might not work on MSVC, so no PR.

**Comment by skrax** [comment]
Fixed since #68
